tests/lib/rules/no-invalid-this.js

```diff
@@ -33,7 +33,7 @@ function NORMAL() {
  * @returns {void}
  */
 function USE_STRICT(pattern) {
-    pattern.code = "\"use strict\"; " + pattern.code;
+    pattern.code = `"use strict"; ${pattern.code}`;
```

Comment:
Wanted to call out @not-an-aardvark's fix for unescaping quotes here - nice job!